cli-wrap/src/main/resources/cli/da-cli.sh

```diff
@@ -23,7 +23,7 @@ printUsage() {
     echo "    $0 list black"
     echo "        List all artifacts in blacklist."
     echo ""
-    echo "    $0 list white [GROUP_ID:ARTIFACT_ID:VERSION]"
+    echo "    $0 list white [PRODUCT_NAME:VERSION]"
     echo ""
     echo "        Each artifact in the white list is associated to a product."
     echo "        List all artifacts and its associated product in the white list."
```

Comment:
Maybe put this change into solo commit (together with the previous one).

Also the message for this option isn't consistent. There is " optionally specify which GAV to show" which is not true.